src/Calendar/Navigation.spec.jsx

```diff
@@ -474,21 +474,28 @@ describe('Navigation', () => {
     const date = new Date(2017, 0, 1);
     const label = 'Custom label';
     const view = 'month';
+    const locale = 'de-DE';
 
     const navigationLabel = jest.fn().mockReturnValue(label);
 
     const component = shallow(
       <Navigation
         {...defaultProps}
         activeStartDate={date}
+        locale={locale}
         navigationLabel={navigationLabel}
         view={view}
       />,
     );
 
     const drillUp = component.find('.react-calendar__navigation__label');
 
-    expect(navigationLabel).toHaveBeenCalledWith({ date, view, label: 'January 2017' });
+    expect(navigationLabel).toHaveBeenCalledWith({
+      locale,
+      date,
+      view,
+      label: 'Januar 2017',
```

Comment:
there is a typo, wonder how test passed.

Reply:
Whoa you had me doubting myself for a second there.

I assume you're saying "Januar" is the typo?

That's actually how you spell January in German, which is what this test is testing.

Reply:
My bad. I should have realized it.
Although It does fail locally for me.